packages/lumx-react/src/utils/type.ts

```diff
@@ -91,3 +91,35 @@ export const isComponentType = (type: ReactElement['type']) => (node: ReactNode)
  * (excluding `NaN` as it can't be distinguished from `number`)
  */
 export type Falsy = false | undefined | null | 0 | '';
+
+/**
+ * Allow at least one of aria-label or arial-labelledby props to be set.
+ */
+export type WithAriaLabelOrLabelledBy =
```

Comment:
We should name it more like `HasTheme`
```suggestion
export type HasAriaLabelOrLabelledBy =
```

Reply:
Sure thing ! Changed :)